src/Sulu/Bundle/MediaBundle/Resources/public/js/components/area-selection/demo.html

```diff
@@ -0,0 +1,81 @@
+<!doctype html>
```

Comment:
No I just wanted to leave the file in till you finished your review, as otherwise you would have no possibility to run the code.